core/navigation/src/main/kotlin/nl/q42/template/navigation/viewmodel/RouteNavigator.kt

```diff
@@ -43,6 +44,7 @@ class MyRouteNavigator : RouteNavigator {
 
     @VisibleForTesting
     fun navigate(state: NavigationState) {
+        Napier.i { state.toString() }
```

Comment:
Why do you want to log this as info? AFAIK this will also be shown in production builds.
What do you think about using Napier.d?

Reply:
I think this is very useful for production, it is as close to an automated reproduction path as you are gooing to get. Note that only error level is logged as a non-fatal exception. lower levels are `crashlytics.log()`ed, which means that they will be added as breadcrumbs to the next exception that is logged

Reply:
I think this addressses your concern https://github.com/Q42/Template.Android/pull/23#discussion_r1469879443